src/group/stabchain/moved_point_selector.rs

```diff
@@ -22,7 +24,7 @@ where
     P: Permutation,
 {
     fn moved_point(&mut self, p: &P) -> usize {
```

Comment:
I think the moved point selector trait will need some love soon. I think it might need to take an additional layer argument and possibly return an option

Reply:
I agree, especially with trying to make it more generic.